Source/WebInspectorUI/UserInterface/Views/FontVariationDetailsSectionRow.css

```diff
@@ -42,16 +42,25 @@
     color: var(--text-color-secondary);
 }
 
-.details-section > .content > .group > .row.font-variation > .value {
+.details-section > .content > .group > .row.font-variation > input[type="number"] {
     grid-column: axis-value;
-    text-align: end;
+    height: max-content;
+    width: 6ch;
+    padding-block: 4px;
+    padding-inline: 0.5ch;
```

Comment:
Does this look odd if you instead just set the padding to be 4px all around? 0.5ch is an odd padding value in my opinion...

Reply:
Yes, I'll revert to using pixels. `ch` didn't work out as I expected.